Program.cs

```diff
@@ -33,11 +43,14 @@ static void Main(string[] args)
 
     static async Task MainAsync(string[] args)
     {
+        var logFactory = new LoggerFactory().AddSerilog();     
```

Comment:
Consider condensing by removing declaration; move computation to assignment at ln 53